src/main/java/com/onlinebookstore/onlinebookstore/model/User.java

```diff
@@ -31,4 +41,50 @@ public class User {
     private String shippingAddress;
     @Column(nullable = false)
     private boolean isDeleted = false;
+    @ManyToMany(fetch = FetchType.EAGER)
+    @JoinTable(
+            name = "user_roles",
+            joinColumns = @JoinColumn(name = "user_id"),
+            inverseJoinColumns = @JoinColumn(name = "role_id")
+    )
+    private Set<Role> roles = new HashSet<>();

```

Comment:
I am not sure that we need init hash set there. Because we can add it using setter

Reply:
Will think about this.

Reply:
yes you were correct